miden/tests/integration/exec_iters.rs

```diff
@@ -0,0 +1,103 @@
+use super::build_test;
+use processor::VmState;
+use vm_core::Felt;
+
+// EXEC ITER TESTS
+// =================================================================
```

Comment:
Could we add one more test which would test the behavior of `fmp`? e.g., a script with a single procedure in it such that the procedure requires 1 local variable and writes to it.

Reply:
fixed